app/scripts/views/config.js

```diff
@@ -11,7 +11,7 @@ export default React.createClass({
 
   getInitialState: function () {
     return {
-      body: JSON.stringify(this.props.config, null, '  '),
+      body: JSON.stringify(this.props.config, null, 2),
```

Comment:
This just blew my mind a little bit, never knew you could specify a number instead of a string.